test/functional/http/post/5.dapps.js

```diff
@@ -474,7 +474,7 @@ describe('POST /api/transactions (type 5) register dapp', function () {
 	describe('unconfirmed state', function () {
 
 		it('duplicate submission identical app should be ok and only last transaction to arrive should be confirmed', function () {
-			transaction = node.lisk.dapp.createDapp(account.password, null, dappDuplicate);
+			transaction = node.lisk.dapp.createDapp(account.password, null, dappDuplicate, -1);
```

Comment:
why we add this?

Reply:
That test has an accurate performance using offset -1. I've realised it while adding offset tests for tx type 0.